physics/Interstitials/UFS_SCM_NEPTUNE/GFS_MP_generic_post.meta

```diff
@@ -430,7 +430,7 @@
   dimensions = (horizontal_loop_extent,vertical_layer_dimension,number_of_tracers)
   type = real
   kind = kind_phys
-  intent = inout
+  intent = in
```

Comment:
no change for save_q

Reply:
The fortran code has this as intent(in). Change is needed here to match